databind/src/main/java/gov/nist/secauto/metaschema/databind/io/json/MetaschemaJsonReader.java

```diff
@@ -394,7 +394,14 @@ public Object readChoiceGroupItem(Object parentItem, IBoundInstanceModelChoiceGr
       JsonParser parser = getReader();
       ObjectNode node = parser.readValueAsTree();
 
-      JsonNode descriminatorNode = ObjectUtils.requireNonNull(node.get(instance.getJsonDiscriminatorProperty()));
+      String descriminatorProperty = instance.getJsonDiscriminatorProperty();
+      JsonNode descriminatorNode = node.get(descriminatorProperty);
+      if (descriminatorNode == null) {
+        throw new IllegalArgumentException(String.format(
+            "Unable to find descriminator property '%s' for object at '%s'.",
+            descriminatorProperty,
+            JsonUtil.toString(parser)));
+      }
```

Comment:
Minor, non-blocking: its discriminator, right?

```suggestion
      String discriminatorProperty = instance.getJsonDiscriminatorProperty();
      JsonNode discriminatorNode = node.get(discriminatorProperty);
      if (discriminatorNode == null) {
        throw new IllegalArgumentException(String.format(
            "Unable to find discriminator property '%s' for object at '%s'.",
            discriminatorProperty,
            JsonUtil.toString(parser)));
      }
```

Reply:
Sadness... Thanks!